### PR TITLE
`GradleInspector` project type fixes

### PR DESCRIPTION
**Pull request by sschuberth**
Please have a look at the individual commit messages for the details.

### CODE REVIEW

**Review by fviernau — APPROVED** [review]

**Comment by fviernau** [comment]
I'm ok with this change, IMO it'd be nice to have another core dev approval for this.

**Review by MarcelBochtler — APPROVED** [review]

**Pull request merged**
